src/rules/function-url-no-scheme-relative/index.js

```diff
@@ -0,0 +1,37 @@
+import {
+  functionArgumentsSearch,
+  isStandardSyntaxUrl,
+  report,
+  ruleMessages,
+  validateOptions,
+} from "../../utils"
+import _ from "lodash"
+
+export const ruleName = "function-url-no-scheme-relative"
+
+export const messages = ruleMessages(ruleName, {
+  rejected: "Unexpected scheme-relative url",
+})
+
+export default function (actual) {
+  return (root, result) => {
+    const validOptions = validateOptions(result, ruleName, { actual })
+    if (!validOptions) { return }
+
+    root.walkDecls(function (decl) {
+      functionArgumentsSearch(decl.toString().toLowerCase(), "url", (args, index) => {
+        const url = _(args).chain().trim().trim("\'").trim("\"").value()
```

Comment:
Feel free to use `_.trim()`, I'm just saying there's no need for the additional complication of `_.chain()`, because it does not add value in this situation.